### ISSUE TITLE
Support for CPU?

### ISSUE THREAD

**Issue by DrewThomasson** [astramind-ai/Auralis#34]
I appear to have trouble running on x86 linux computers with no GPU is that possible?

**Comment by mlinmg** [comment]
yes by default we use some gpu only option, we will add cpu only support
